packages/mobile/src/geth/network-config.ts

```diff
@@ -18,4 +18,12 @@ export default {
     nodeDir: `.${Testnets.alfajores}`,
     syncMode: SYNC_MODE_ULTRALIGHT,
   },
+  [Testnets.pilot]: {
```

Comment:
(not for this PR)

@cmcewen How about we remove `network-config.ts` completely at this point? We just need a list of known networks.

Reply:
sounds great 👍